apps/docs/src/app/core/component-docs/busy-indicator/busy-indicator-docs.component.html

```diff
@@ -1,4 +1,4 @@
-<fd-docs-section-title id="default" componentName="BusyIndicator">
+<fd-docs-section-title id="default" componentName="busyIndicator">
```

Comment:
not consistent with the others which are using kebab case. For example: `fixed-card-layout`.